tests/test_cli.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+import os
+import signal
+import threading
+import time
+
+from prometheus_pgbouncer_exporter import cli
+from unittest import mock
+
+
+CURR_DIR=os.path.dirname(os.path.realpath(__file__))
+
+
+class TestCLI(unittest.TestCase):
+
+    def testReadConfigFile(self):
+
+        # should read and validate this config file without any issues
+        config = cli.read_config_file(CURR_DIR + '/fixtures/config-with-one-pgbouncer.yml')
+
+        self.assertEqual(config.getExporterHost(), "0.0.0.0")
+        self.assertEqual(config.getExporterPort(), 1234)
+        self.assertEqual(len(config.getPgbouncers()), 1)
+
+        self.assertEqual(config.getPgbouncers()[0].getDsn(), "postgresql://user:password@host:6431/pgbouncer")
+        self.assertEqual(config.getPgbouncers()[0].getDsnWithMaskedPassword(), "postgresql://user:***@host:6431/pgbouncer")
+        self.assertEqual(config.getPgbouncers()[0].getConnectTimeout(), 2)
+        self.assertEqual(config.getPgbouncers()[0].getIncludeDatabases(), ["one", "two"])
+        self.assertEqual(config.getPgbouncers()[0].getExcludeDatabases(), ["three"])
+        self.assertEqual(config.getPgbouncers()[0].getExtraLabels(), {"first": "1", "second": "2"})
+
+        # Should raise exception saying 'Cannot read config file'
+        with self.assertRaises(Exception) as err:
+            cli.read_config_file(CURR_DIR + '/path/to/none.yml')
+
+        self.assertEqual('Cannot read config file', str(err.exception))
+
+        # Should raise exception saying 'Invalid config file'
+        with self.assertRaises(Exception) as err:
+            cli.read_config_file(CURR_DIR + '/fixtures/config-empty.yml')
+
+        self.assertEqual('Invalid config file', str(err.exception))
+
+    @mock.patch('prometheus_pgbouncer_exporter.cli.os._exit')
+    @mock.patch('prometheus_pgbouncer_exporter.collector.PgbouncersMetricsCollector.update')
+    @mock.patch('prometheus_pgbouncer_exporter.collector.PgbouncersMetricsCollector.collect')
+    @mock.patch('prometheus_pgbouncer_exporter.cli.argparse.ArgumentParser')
+    def testSignalHandling(self, _argparser, _collect, _update, _exit):
```

Comment:
Running this test is verbose and that's something we should avoid. Either you increase the `_args.log_level` to `CRITICAL` or you mock the logger in case you want to capture the call (not needed in this case).

Also, we don't use variables or parameters with an underscore prefix `_`, so you should rename them to remove it